docs/painless/painless-getting-started.asciidoc

```diff
@@ -119,6 +120,31 @@ GET hockey/_search
 ----------------------------------------------------------------
 // CONSOLE
 
+[float]
+===== Missing values
+
+Currently by default, if a document is missing a numeric field `field`,
+`doc['field'].value` returns `0` for this document. This default behaviour
+will be changed in the next major version of elasticsearch:
+if a document is missing a field `field`, `doc['field']` for this document
+will return `null`. You can set a system property
+`export ES_JAVA_OPTS="-Des.script.null_for_missing_value=true"' on a node
+to make this node's behaviour compatible with the future major version.
+Otherwise, every time the node starts a deprecation warning will remind you
+about this forthcoming change.
+
+
+===== Multiple values
```

Comment:
I'm not sure that the getting started guide is the place for either of these things to be documented. Maybe @debadair has a better suggestion for where they could fit in?